contrib/single-node.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+CHAINID=$1
+GENACCT=$2
+
+if [ -z "$1" ]; then
+  echo "Need to input chain id..."
+  exit 1
+fi
+
+if [ -z "$2" ]; then
+  echo "Need to input genesis account address..."
+  exit 1
+fi
+
+# Build genesis file incl account for passed address
+coins="10000000000stake,100000000000samoleans"
+gaiad init --chain-id $CHAINID $CHAINID
+gaiacli keys add validator --keyring-backend="test"
+gaiad add-genesis-account validator $coins --keyring-backend="test"
+gaiad add-genesis-account $GENACCT $coins --keyring-backend="test"
+gaiad gentx --name validator --keyring-backend="test"
+gaiad collect-gentxs
+
+# Set proper defaults and change ports
+sed -i 's/"leveldb"/"goleveldb"/g' ~/.gaiad/config/config.toml
```

Comment:
This seems unnecessary, the default should already be "goleveldb".

Reply:
ok, well its working, lets just leave it for now?

Reply:
I've fixed this